pkg/deleter/deleter.go

```diff
@@ -336,15 +336,15 @@ func (d *Deleter) execScript(pvName string, blkdevPath string, exe string, exeAr
 
 // runJob runs a cleaning job.
 // The advantages of using a Job to do block cleaning (which is a process that can take several hours) is as follows
-// 1) By naming the job based on the specific name of the volume, one ensures that only one instance of a cleaning
```

Comment:
This change doesn't look like it's part of the CVE changes

Reply:
It is to fix the gofmt issue.